btree/btree.c

```diff
@@ -8411,7 +8417,7 @@ M0_INTERNAL int64_t m0_btree_lrulist_purge(int64_t size)
 		prev      = ndlist_tlist_prev(&btree_lru_nds, node);
 		if (node->n_txref == 0 && node->n_ref == 0) {
 			curr_size = node->n_size;
-			seg       = node->n_tree->t_seg;
+			seg       = node->n_seg;
 			rnode     = segaddr_addr(&node->n_addr);
 
 			rc = unmap_node(rnode, curr_size);
```

Comment:
You need to get the BE allocator lock before unmapping this memory.

Reply:
Done